adhocteam/Head-Start-TTADP
# Context textarea overlapping goals multiselect

## Description of change
When creating a new goal using the multiselect component, it's possible for the context textarea to partially overlap the dropdown options, making it hard or impossible to create a new goal, since that will be the last option.

## How to test
Create a new goal from a region with several extant goals. You should not see the text entered in the context area through the menu, and you should be able to create a new goal.

## Issue(s)

## Checklists

### Every PR


- [n/a] Meets issue criteria
- [n/a] JIRA ticket status updated
- [n/a] Code is meaningfully tested
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
